examples/mp-kv-sharding/src/main.rs

```diff
@@ -752,6 +1046,24 @@ fn main() {
     tests.add("NODE ADDED", test_node_added, config);
     tests.add("NODE REMOVED", test_node_removed, config);
     tests.add("NODE REMOVED AFTER CRASH", test_node_removed_after_crash, config);
+    tests.add(
+        "MODEL CHECKING",
+        test_model_checking_normal,
```

Comment:
Предлагаю везде сократить имена функций с `model_checking` до `mc`: 

```suggestion
        test_mc_normal,
```